modules/picard/liftovervcf/main.nf

```diff
@@ -0,0 +1,46 @@
+
+process PICARD_LIFTOVERVCF {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::picard=2.26.10" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:2.26.10--hdfd78af_0' :
+        'quay.io/biocontainers/picard:2.26.10--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(input_vcf)
+    path dict
+    path chain
+    path fasta
+
+    output:
+    tuple val(meta), path("*lifted.vcf")  , emit: lifted
```

Comment:
It is recommended to have a compressed file as output. Is that possible here?

Reply:
Fixed :)